src/components/container/SurveyTaskList/SurveyTaskList.css

```diff
@@ -1,3 +1,7 @@
+.mdhui-survey-task-list{
```

Comment:
This isn't a view but a component; it should not have a background

Reply:
I have a PR open so you can customize view background colors which should allow you to achieve this in the view builder